facdb/geocode/function1A.py

```diff
@@ -0,0 +1,88 @@
+import json
```

Comment:
Do not need a 1A call

Reply:
Also, @td928 why not use the lat and long fields to create the geometry?  See screenshot below of how to convert the lat/long values
![Screen Shot 2022-05-09 at 4 55 30 PM](https://user-images.githubusercontent.com/5611960/167496973-358aacb4-5283-4503-8d3a-b86b9d2c919f.png)

Reply:
My didn't really understood arp lat long and google did not give me good results. @AmandaDoyle Please let me know if you could give me a tutorial or point me to a resource to explain how to use them. And yeah it would potentially be a much easier solution.

Reply:
https://geodesy.noaa.gov/AERO/arpcomp/arpframe.html

Reply:
I think its pretty complex measurement that pilots must use for navigation

Reply:
Can you just use the geometry?

Reply:
this geom is created from geocoding not in source data @mbh329